code/obj/item/tank.dm

```diff
@@ -9,6 +9,8 @@ Contains:
 -Plasma Tank
 */
 
+#define TANK_VOLUME_LITRES 70 //! The volume of a normal tank in litres
```

Comment:
would like to see `LITERS` added into `_units.dm` would also prefer using the American spelling for consistency

Reply:
Dangit im american and i still used litres. how could i let this happen to myself

Reply:
I did this in #20200 but in light of that PR basically getting left to collect dust I guess I could just put it here.. but 🤷‍♀️